demos/storybook/stories/ hero/with-full-config.stories.ts

```diff
@@ -13,6 +13,8 @@ export const withFullConfig = (): any => ({
         value: text('value', '470'),
         units: text('units', 'RPM'),
         showSecondary: boolean('Show Secondary Icon', true),
-        iconColor: color('primary.style.color', Colors.blue[500]),
+        iconSize: number('iconSize', 98),
```

Comment:
should we just limit this to large and normal?

Reply:
I'm going to update to the iconSize to only support two values 'normal' and 'large'.  If a user wants a custom size they can override it via sass.